App/UI/GreenCertificate/ModalDetail/GreenCertificateVaccineDetail/GreenCertificateVaccineDetailView.swift

```diff
@@ -17,755 +17,797 @@ import Models
 import Tempura
 
 struct GreenCertificateVaccineDetailVM: ViewModelWithLocalState {
-    
-    let greenCertificate: GreenCertificate
+  let greenCertificate: GreenCertificate
 }
 
-extension GreenCertificateVaccineDetailVM {
-    init?(state: AppState?, localState : GreenCertificateVaccineDetailLS) {
-        guard let _ = state else {
-            return nil
-        }
+struct configurationStateVaccine {
+  static var _state = ["": ["": ""]]
+}
 
-        self.greenCertificate = localState.greenCertificate
+extension GreenCertificateVaccineDetailVM {
+  init?(state: AppState?, localState: GreenCertificateVaccineDetailLS) {
+    guard let _ = state else {
+      return nil
     }
+
+    self.greenCertificate = localState.greenCertificate
+    configurationStateVaccine._state = (state?.configuration.eudccExpiration)!
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/UI/GreenCertificate/ModalDetail/GreenCertificateVaccineDetail/GreenCertificateVaccineDetailView.swift;
  Line: 34;
-->

- :warning: swiftlint (0.39.2): Force unwrapping should be avoided. (force_unwrapping)